src/create-card.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+
+const styles = {
+  background: {
+    width: '32rem'
+  }
+}
+
+export default class CreateCard extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSave = this.handleSave.bind(this)
+  }
+  handleSave(e) {
+    e.preventDefault()
+    const { question, answer } = this.state
+    if (!question || !answer) {
+      alert('Please enter content in both fields')
+    }
+    else {
+      const flashCards = JSON.parse(localStorage.getItem('flashCards')) || []
```

Comment:
Instead of storing these in `localStorage`, you need to get them back up to `App` state so when we go to show the list of cards, they're already there.